sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -126,6 +133,15 @@ internal open class KlaviyoApiRequest(
             }
         }
 
+    /**
+     * Tracks number of attempts to limit retries
+     */
+    final override var attempts = 0
```

Comment:
should the default be 1?

Reply:
Defaults to 0 and always increments before send (there's a test for this). That allows distinguishing a totally unsent request from first attempt actually pending.